modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/security/permission/MBCategoryPermission.java

```diff
@@ -1,65 +0,0 @@
-/**
```

Comment:
@dantewang @yuhai
I am not going to merge this commit.

Reason is the published OSGi service is tracking by a service tracker, and this may have a side effect on runtime.

Theoretically this should be safe to remove, but let's delay this to second round, and make the first round simple and clean.

Tina.